hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSConfigToCSConfigConverter.java

```diff
@@ -214,31 +216,31 @@ public void testDefaultMaxAMShareDisabled() throws Exception {
 
     converter.convert(params);
 
-    Configuration conf = converter.getCapacitySchedulerConfig();
+    CapacitySchedulerConfiguration conf = converter.getCapacitySchedulerConfig();
 
     // -1.0 means disabled ==> 1.0 in CS
     assertEquals("Default max-am-resource-percent", "1.0",
         conf.get(CapacitySchedulerConfiguration.
             MAXIMUM_APPLICATION_MASTERS_RESOURCE_PERCENT));
 
-    // root.admins.bob -1.0 equals to the default -1.0
-    assertNull("root.admins.bob maximum-am-resource-percent should be null",
-        conf.get(PREFIX + "root.admins.bob.maximum-am-resource-percent"));
+    // root.admins.bob is unset,so falling back to the global value
+    assertEquals("root.admins.bob maximum-am-resource-percent", 1.0f,
+        conf.getMaximumApplicationMasterResourcePerQueuePercent("root.admins.bob"), 0.0f);
 
     // root.admins.alice 0.15 != -1.0
-    assertEquals("root.admins.alice max-am-resource-percent", "0.15",
-        conf.get(PREFIX + "root.admins.alice.maximum-am-resource-percent"));
+    assertEquals("root.admins.alice max-am-resource-percent", 0.15f,
+        conf.getMaximumApplicationMasterResourcePerQueuePercent("root.admins.alice"), 0.0f);
 
-    // root.users.joe is unset, inherits -1.0
-    assertNull("root.users.joe maximum-am-resource-percent should be null",
-        conf.get(PREFIX + "root.users.joe.maximum-am-resource-percent"));
+    // root.users.joe is unset,so falling back to the global value
+    assertEquals("root.users.joe maximum-am-resource-percent", 1.0f,
+        conf.getMaximumApplicationMasterResourcePerQueuePercent("root.users.joe"), 0.0f);
   }
 
   @Test
   public void testConvertACLs() throws Exception {
     converter.convert(config);
 
-    Configuration conf = converter.getCapacitySchedulerConfig();
+    CapacitySchedulerConfiguration conf = converter.getCapacitySchedulerConfig();
 
     // root
     assertEquals("root submit ACL", "alice,bob,joe,john hadoop_users",
```

Comment:
There is a getAcl() method in the config class which you can use in this test case.